hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/utils/FederationStateStoreFacade.java

```diff
@@ -513,6 +525,32 @@ public Map<String, SubClusterPolicyConfiguration> invoke(
     return cacheRequest;
   }
 
+  @SuppressWarnings("unchecked")
+  private Object buildGetApplicationHomeSubClusterRequest(String appId) {
```

Comment:
Can we cast it here?

Reply:
Thanks for your suggestion, I will modify the code.